_posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Snowplow Objective-C Tracker 1.0.0 released"
+title-short: Snowplow Objective-C Tracker 1.0.0
+tags: [snowplow, mobile, obj-c, ios]
+author: Mike
+category: Releases
+permalink: /blog/2019/02/20/snowplow-ios-tracker-1.0.0-released/
+discourse: true
+---
+
+We are pleased to announce a new release of the [Snowplow Objective-C Tracker][ios-tracker]. [Version 1.0.0][tracker-100] brings dependency updates, some bug fixes and support for integrating the tracker into your project with Carthage.
+
+Read on below the fold for:
+
+1. [Carthage support](#carthage-support)
+2. [Other changes](#updates)
+3. [Upgrading](#upgrade)
+4. [Documentation and help](#doc)
+
+<!--more-->
+
+<h2 id="carthage-support">1. Carthage support</h2>
+
+If you are already using Carthage, just add this line to your `Cartfile`:
+
+```
+# Snowplow tracker
+github "snowplow/snowplow-objc-tracker"
+```
+
+For more information on Carthage, see their website [here](https://github.com/Carthage/Carthage#quick-start).
+
+<h2 id="updates">2. Other changes</h2>
+
+Other updates and fixes include:
+
+* Exposes session property in SPTracker ([#256][256])
+* Fix parameter documentation ([#389][389])
+* Make initializers unavailable in SPEmitter and SPTracker ([#255][255])
+* Add OpenIdfa files to framework target ([#382][382])
+* Resolve potential memory leaks from implicit retain of self ([#353][353])
+* Embed Swift standard library for tests ([#388][388])
+* Disable always embed Swift standard libraries ([#381][381])
+* Remove exceptions ([#383][383])
+* Explicitly close all statements in FMDB ([#384][384])
+* Fix getDocuments function in consent events ([#386][386])
+* Add support for Carthage ([#291][291])
+* Replace Reachability with Reachability.swift ([#385][385])
+
+Many thanks to [Kevin Malek][kevmalek], [Miguel Angel Quinones Garcia][DarthMike], [Sébastien Duperron][sduperron-viadeo], and [Stewart Gleadow][sgleadow] for their contributions!
+
+<h2><a name="upgrading">3. Upgrading</a></h2>
+
+To add the Snowplow Objective-C Tracker as a dependency to your own app, add the following into your `Podfile`:
+
+{% highlight ruby %}
+pod 'SnowplowTracker', '~> 1.0'
+{% endhighlight %}
+
+If you are using Carthage, just add this line to your `Cartfile`:
+
+{% highlight bash %}
+# Snowplow tracker
+github "snowplow/snowplow-objc-tracker"
+{% endhighlight %}
+
+<h2><a name="help">4. Getting help</a></h2>
+
+As always, please check out the following links:
+
+* The [technical documentation][tech-docs]
```

Comment:
Does this release not include the documentation refresh Mike?

Reply:
Updated already with highlights on the documentation